src/containers/Nav.jsx

```diff
@@ -21,27 +22,41 @@ class ActiveLink extends Component {
   }
 
   open (e) {
+    e.preventDefault()
     this.setState({ opening: true })
     this.props.onClick()
-      .then(() => this.setState({ opening: false }))
+      .then(() => {
+        this.setState({ opening: false })
+        this.props.router.push(this.props.to)
+      })
   }
 
-  render ({ to, className, activeClassName, children }, { opening }) {
+  render () {
+    const { router, location, to, activeClassName, children, ...props } = this.props
+    const { opening } = this.state
+
+    props.href = router.createHref(to)
```

Comment:
I needed a different behavior than the standard's `Link` (the `setState` call that displays the spinner)